### PR TITLE
Feature 898 headers

### LINKED ISSUE #898: Documentation: Make Headers Consistent in METplus components User's Guides
## Describe the Task ##
The headers for the METplus components User's Guides may not be consistent, but we want them to be. 

Here is the syntax from the RST docs (https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections):

\# with overline, for parts (e.g. the RTD documentation for the component (i.e. docs/index.rst), each guide (e.g. User's Guide, Contributor's Guide) (i.e. docs/Users_Guide/index.rst, docs/Contributors_Guide/index.rst))
\* with overline, for chapters
=, for sections
-, for subsections
^, for subsubsections
", for paragraphs

### Time Estimate ###
1 to 3 days of work

### Sub-Issues ###
Consider breaking the task down into sub-issues.
- [x] Start with the METcalcpy installation documentation: METcalcpy/docs/Users_Guide/installation.rst in the repository
        ([Documentation: Make Headers Consistent in METcalcpy User's Guides #313](https://github.com/dtcenter/METcalcpy/issues/313))
- [x] METplotpy - Lisa:They were all correct except Chapter 21 tcmw_cross_section.rst. 7/25/23 pull request submitted https://github.com/dtcenter/METplotpy/issues/371.  Julie found underlines that needed to be dashes.  Pull request approved 7/26/23 with changes.
- [x] METplus -   Please review "All" guides, including, 
       - [x] User's Guide (Please note, the usecases.rst file pulls converted text from python files located under the "use_cases" folder. These do not follow the standard protocol and are separate from all other forms of the documentation. It does not need to be reviewed.)
       - [x] Contributor's Guide (work completed 9/4/23), 
       - [x] Verification Datasets Guide - Julie: Lisa, you should still check every file to be detailed and thorough, but upon a quick glance I noticed that all of the Verification Dataset chapters have incorrect headers of underlining ===, but should be underlined and overlined ****. For this work, you can just use this existing issue for METplus for your feature branch.
       - [x] Release Guide, you may need help with this.  Lisa: This was very tricky and confusing. I created a sheet, [2023 Release Guide adventures](https://docs.google.com/spreadsheets/u/0/d/1LttGC1Xlw-2JEaQo7hMv8QdsyzjvvZ3_Xz4SvB98hC8/edit) to try and keep track of the 100 files linked across many different chapters.
       - [x] Lisa: I had questions about things outside of the scope of work.  I've captured them here for review: [2023 METplus oddities discovered while cleaning up header consistency](https://docs.google.com/document/u/0/d/13nvFrJoGVuUVngHiI9IGwb3vwO2ZStvG_16cgFHseo0/edit) Per Julie, it is okay to pass on these issues since they are outside of the scope of work.
- [x] MET (Lisa, you need to create a [Sub-Issue in the MET repository](https://github.com/dtcenter/MET/issues/new?assignees=&labels=alert%3A+NEED+ACCOUNT+KEY%2C+alert%3A+NEED+MORE+DEFINITION%2C+alert%3A+NEED+CYCLE+ASSIGNMENT%2C+type%3A+sub-issue&projects=&template=sub-issue.md&title=) for this work; Please use the following title **Documentation: Make Headers Consistent in All MET Guides**) By "All" guides, that means the 
       - [x] User's Guide,
       - [x] Contributor's Guide
   The above items were completed on 11/9/23.
- [x] In creating the description for the new issue please use the METcalcpy issue link above as an example.   MET issue created 10/16/23: https://github.com/dtcenter/MET/issues/2716
- [x] METdatio (Lisa, you need to create a [Sub-Issue in the METdataio repository](https://github.com/dtcenter/METdataio/issues/new?assignees=&labels=alert%3A+NEED+ACCOUNT+KEY%2C+alert%3A+NEED+MORE+DEFINITION%2C+alert%3A+NEED+CYCLE+ASSIGNMENT%2C+type%3A+sub-issue&projects=&template=sub-issue.md&title=) for this work; Please use the following title **Documentation: Make Headers Consistent in All METdataio Guides**) By "All" guides, that means the User's, Contributor's, etc. guides. In creating the description for the new issue please use the METcalcpy issue link above as an example.  Issue created 10/16/23: https://github.com/dtcenter/METdataio/issues/238. Work completed and pull request closed 10/19/23.
- [x] METviewer (Lisa, you need to create a [Sub-Issue in the METviewer repository](https://github.com/dtcenter/METviewer/issues/new?assignees=&labels=alert%3A+NEED+ACCOUNT+KEY%2C+alert%3A+NEED+MORE+DEFINITION%2C+alert%3A+NEED+CYCLE+ASSIGNMENT%2C+type%3A+sub-issue&projects=&template=sub-issue.md&title=) for this work; Please use the following title **Documentation: Make Headers Consistent in All METviewer Guides**) By "All" guides, that means the User's, Contributor's, etc. guides. In creating the description for the new issue please use the METcalcpy issue link above as an example.  Issue createdd 10/16/23: https://github.com/dtcenter/METviewer/issues/497 

### Relevant Deadlines ###
None

### Funding Source ###
2783543 - Lisa has hours on this account.

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Review **projects** and select relevant **Repository** and **Organization** ones or add "alert:NEED PROJECT ASSIGNMENT" label
- [x] Select **milestone** to next major version milestone or "Future Versions"

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdatadb](https://github.com/dtcenter/METdatadb/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Task Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)**, **Project(s)**, **Milestone**, and **Linked issues**
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by lisagoodrich**
## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br> This was a big adventure.  All of the headers for the different sections: Contributor's Guide, User's Guide, Release Guide and Verification Dataset Guide have been updated.
In some cases, the text within the headers has been capitalized for consistency.
Three changes were made in the glossary.rst file to update notes to warnings for consistency: POINT_STAT_GRID
POINT_STAT_POLY
PB2NC_POLY
I compared the feature_898_headers branch  https://metplus.readthedocs.io/en/feature_898_headers/Users_Guide/index.html
to the development branch in ReadTheDocs https://metplus.readthedocs.io/en/develop/Users_Guide/index.html. I did this for ALL of the above sections.
In some cases, I opened up the files in GitHub to confirm that the new and correct headers were being used. https://github.com/dtcenter/METplus/tree/feature_898_headers/docs

The only area where I didn't check the headers was here: https://metplus.readthedocs.io/en/feature_898_headers/generated/met_tool_wrapper/index.html and here: https://metplus.readthedocs.io/en/feature_898_headers/generated/model_applications/index.html
I do not know where these files are being generated.

REMINDER: Please keep the issue open.  The MET headers need to be finished before this issue can close.

- [X] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
I feel very confident that these headers are correct.  However, this was a tedious and complex process.  I really struggled on the Release Guide. There are over 100 files that link to many different directories.  This formatting also assumes the file sections will be at the SAME level across different tools.  So far, this is the case.
Please do a spot check of the headers as you see fit.
George asked to be included on this pull request. Since is working in METplus.

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes, there are no errors.]**

- [x] Do these changes include sufficient testing updates? **[Yes]**

- [x] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Please complete this pull request review by **11/3/23**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements) table.
- [x] Review the source issue metadata (required labels, projects, and milestone).
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization** level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [x] After submitting the PR, select the :gear: icon in the **Development** section of the right hand sidebar. Search for the issue that this PR will close and select it, if it is not already selected.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: docs/Users_Guide/installation.rst [hunk 1]
```diff
@@ -277,7 +277,7 @@ GFDL Tracker (optional)
     -  Instructions on how to configure and use the GFDL tracker are found here
        https://dtcenter.org/sites/default/files/community-code/gfdl/standalone_tracker_UG_v3.9a.pdf
 
-Disable UserScript wrapper (optional)
+Disable UserScript Wrapper ()ptional)
```
Comment (georgemccabe) [inline docs/Users_Guide/installation.rst]:
```suggestion
Disable UserScript Wrapper (Optional)
```

**Comment by georgemccabe** [comment]
I only reviewed the User's Guide content to make sure there are no conflicts with my changes since I am working on those files. I don't see anything of concern, but I did find a minor typo (see suggested change).

File: docs/Contributors_Guide/add_use_case.rst [hunk 2]
```diff
@@ -183,7 +183,7 @@ Inside this file add the following each on a single line:
 * A short description of the category
 
 For example,
-*docs/use_cases/model_applications*/**air_quality_and_comp/README.rst**
+*docs/use_cases/model_applications/air_quality_and_comp/README.rst*
```
Comment (jprestop) [inline docs/Contributors_Guide/add_use_case.rst]:
```suggestion
*docs/use_cases/model_applications*/**air_quality_and_comp/README.rst**
```
I know this isn't our typical formatting, but in this case it is formatted this way for emphasis on air_quality_and_comp/README.rst so let's leave it as-is.

**Review by jprestop — APPROVED** [review]
@lisagoodrich Thank you for all of your work on this task.  I know it was A LOT.  Thank you also for checking and rechecking.  I have reviewed all of the files and approve this request.  It's ready for squashing and merging whenever you're ready.  And, I know we already talked about it and you have a note about it, but just a reminder to leave the issue open.

**Pull request merged**